vizro-core/src/vizro/models/_controls/filter.py

```diff
@@ -269,6 +271,12 @@ def _get_min_max(targeted_data: pd.DataFrame, current_value=None) -> tuple[float
         _max = targeted_data.max(axis=None)
 
         # Convert to datetime if the column is datetime64
+        # AM question: I think this could be simplified:
+        #  * column type is already stored in _column_type and validated in __call__ with validate_column_type. Hence
+        #    we can just do if instance(self.selector, SELECTORS["temporal"]
```

Comment:
We can declare this as a non-static method and do that.

Reply:
Ah yes, I see it would need to be a non-static method for that. It may or may not be a good idea then.

Reply:
It turns out that we don't have to do this check anymore. Explanation is in the https://github.com/mckinsey/vizro/pull/879#discussion_r1860249153